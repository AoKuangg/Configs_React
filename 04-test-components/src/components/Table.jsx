export default function Table() {
  return (
    <>
        <div>Zzz</div>
        <div>Table</div>
    </>
  )
}
