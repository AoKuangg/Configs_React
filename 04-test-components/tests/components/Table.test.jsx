import { render } from "@testing-library/react";
import Table from "../../src/components/Table";

describe("Captura del componente Tabla.jsx", () => {
  it("Fotico:", () => {
    let { container } = render(<Table />);
    expect(container).toMatchSnapshot();
  });
});
