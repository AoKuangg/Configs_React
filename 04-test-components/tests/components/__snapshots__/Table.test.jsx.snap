// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Captura del componente Tabla.jsx Fotico: 1`] = `
<div>
  <div>
    Zzz
  </div>
  <div>
    Table
  </div>
</div>
`;
